packages/vue-script-setup-converter/src/lib/convertSrc.ts

```diff
@@ -52,7 +56,17 @@ export const convertSrc = (input: string) => {
     sourceFile
       .getStatements()
       .filter((state) => !Node.isExportAssignment(state))
-      .map((x) => x.getText())
+      .map((x) => {
+        if (x.isKind(SyntaxKind.ImportDeclaration)) {
+          if (hasNamedImportIdentifier(x, "defineComponent")) {
+            removeNamedImportIdentifier(x, "defineComponent");
```

Comment:
> (I don't want to rewrite the original sorce.)

Pros.

I thought so too, but chose to rewrite the original once, because no real harm would come from rewriting the original at this time, and because duplicating the importDeclaration would have been somewhat tedious.

However, on second thought, I decided that it could be a breeding ground for bugs in the future, so I will fix it.

😺